Nosenti/certificate-manager---Innocent
# Task7-KAN-38 Multi Language Support

#### PR description
- This PR change languages from English to German and Vice Versa

#### Ticket / Jira Link
- [KAN-38](https://task-force-pro-dccs.atlassian.net/jira/software/projects/KAN/boards/1?assignee=712020%3A67b4d316-bd6f-4b16-a625-69afa0078a10&selectedIssue=KAN-38)

#### Checklist:
- [x] I have created screenshots to show the german version when it is selected

#### How To Test Changes
- In the header of the page, check dropdown of languages
- Switch from English to German and the entire static content of the page will change. 

#### Screenshots:
![image](https://github.com/user-attachments/assets/e9b8e0ed-cdb7-4bee-bfa1-a71ebde90c13)
